testdata/pr_228.ivy

```diff
@@ -0,0 +1,23 @@
+# Argument with prec < conf.floatPrec
+)prec 64
+a = float 2
+)prec 128
+(sqrt a) == sqrt 2
+	1
+
+# Argument with prec > conf.floatPrec
+)prec 256
+a = float 2
+)prec 128
+(sqrt a) == sqrt 2
+	1
+
+# Same test for BigFloat.Sprint
+)prec 64
+a = float 1 << 10000
+)prec 128
+)format "%.30g"
+a
+	1.99506311688075838488374216268e+3010 
+)format ""
```

Comment:
there should be a blank line here, but in any case these last two lines are unnecessary. the test runner resets the state for each test.

Reply:
also, please take a look at the other files in this directory. they aren't organized this way. having a file for the individual test is fine in general, but not consistent with the organization here.

for consistency these tests should just go at the end of unary_bigfloat.ivy (which is missing a trailing newline - my bad) and tagged with the issue number.

perhaps in future there should be a bugs subdirectory or some such, but that shouldn't be part of this change.

Reply:
I did not look into it further at the time, but not resetting precision after the test broke other tests. Precision is indeed not reset in the `reset()` function (format is). There are other things missing: if I insert a `testConf = config.Config{}` at the start of `reset()`, this breaks `debug.ivy`.